server/src/main/java/org/spine3/server/entity/AbstractEntity.java

```diff
@@ -157,12 +160,126 @@ private static IllegalStateException noSuchConstructor(String entityClass, Strin
     }
 
     /**
+     * Obtains constructor for the passed aggregate part class.
```

Comment:
An article before `constructor` is missing.

Also I would additionally explain which constructor is expected, as long as an end-user is able to define a custom one in the descendant classes.